salt/modules/nxos.py

```diff
@@ -338,11 +339,15 @@ def sendline(command, method='cli_show_ascii', **kwargs):
         Hint: White space separated commands should be wrapped by double quotes
         """.format(smethods, method)
         return msg
-
-    if salt.utils.platform.is_proxy():
-        return __proxy__['nxos.sendline'](command, method, **kwargs)
-    else:
-        return _nxapi_request(command, method, **kwargs)
+    try:
+        if salt.utils.platform.is_proxy():
+            return __proxy__['nxos.sendline'](command, method, **kwargs)
+        else:
+            return _nxapi_request(command, method, **kwargs)
+    except socket_error as e:
+        return e.strerror
+    except NxosError as e:
+        return e.strerror
```

Comment:
fyi: you can combine these exceptions:
```
    except (socket_error, NxosError) as e:
        return e.strerror
```

Reply:
Yeah I know.  I intentionally left them separate if we need to handle SSH vs NX-API errors differently for new use cases.  If you feel strongly I can combine them though for now.

Reply:
It's fine. Don't change it.